frontend/src/metabase/static-viz/index.js

```diff
@@ -4,6 +4,6 @@ import StaticChart from "./containers/StaticChart";
 
 export function RenderChart(type, options) {
   return ReactDOMServer.renderToStaticMarkup(
-    <StaticChart type={type} options={options} />,
+    <StaticChart type={type} options={options} colors={options.colors} />,
```

Comment:
Not sure if we need to pass `colors` as an extra prop since it's already in `StaticChart`. I guess you do that since you want to assign a default value to `colors` but I don't think you actually need to do that since both `null` and `undefined` can be destructured into an object just fine.

- https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/frontend/src/metabase/static-viz/containers/StaticChart/StaticChart.tsx#L29-L30
- https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/frontend/src/metabase/static-viz/lib/colors.ts#L4-L5
- ![image](https://user-images.githubusercontent.com/1937582/185309714-81733da3-fd79-4081-b4fd-afb084b66c7c.png)

So, if I'm not mistaken you should be fine dropping default values for both colors props and even remove the `colors` prop from `StaticChart`